### ISSUE TITLE
Safari saves into different places from iframe and directly on site

### ISSUE THREAD

**Issue by gen4sp** [marcuswestin/store.js#172]
I have a widget. The widget could be or opened directly either embed into 3rd party site via iframe. 
So it seems Safari saves (and reads) the data in different places in booth cases. So I can't read in iframe state data saved on site and vice verse. 
Any ideas?

**Comment by marcuswestin** [comment]
This is how browsers are designed - when an iframe loads a page from a different domain, it will store cookies, localstorage, etc tied to that domain. Unless your hosting page the iframe are on the same domain, they won't have access to the same data.

You can still communicate between the two using window.postMessage, so if you want to get data from the iframe page you should send a message via postMessage and then have the iframed page send back a message containing the data (also via postMessage).

In short: not a store.js issue, use postMessage :)

Cheers!
Marcus
